ngraph_bridge/ie_executable.h

```diff
@@ -33,6 +33,8 @@ namespace ngraph_bridge {
 // A Inference Engine executable object produced by compiling an nGraph
 // function.
 class IE_Executable final : public Executable {
+  using TensorMap = unordered_map<string, shared_ptr<ngraph::runtime::Tensor>>;
```

Comment:
May be add a comment specifying what the map is?

Reply:
Isn't it self-explanatory? And this is just a shorthand alias, not even a class member.